synapse/storage/databases/main/cache.py

```diff
@@ -329,6 +329,9 @@ def _invalidate_caches_for_event(
                 "get_forgotten_rooms_for_user", (state_key,)
             )
 
+        if etype == EventTypes.ServerACL:
+            self.hs.get_storage_controllers().state.get_server_acl_for_room.invalidate((room_id,))
```

Comment:
Do we need to worry about telling other HSes to invalidate this cache over replication?

Reply:
This is inside the method which gets called when things get invalidated over replication.

Reply:
Aha. Thanks.

Reply:
(also `s/HSes/workers/`)